elm/src/FaqNurses/Update.elm

```diff
@@ -0,0 +1,153 @@
+      , isVisible = False
+      }
+    , { question = "How should I prepare for the NCLEX?"
+      , answer = [ Paragraph "We provide study materials and resources to help you prepare. We also connect you with other nurses who are also preparing for the test to support each other." ]
+      , isVisible = False
+      }
+    , { question = "What happens if I fail the NCLEX in my first attempt?"
+      , answer = [ Paragraph "Failed tests will prolong the process so it's important to do your best to pass the first time. You may take the test again in 45 days at the earliest. This is a rule set by the testing authority." ]
+      , isVisible = False
+      }
+    , { question = "If I'm already in the immigration and licensing process can I work with Flint?"
+      , answer = [ Paragraph "We can but it depends on the situation. If you have specific questions you can reach out to our team to find out more." ]
+      , isVisible = False
+      }
+    , { question = "What is the level of English proficiency that is required?"
+      , answer = [ Paragraph "Employers care that you can communicate effectively with staff and patients. They require you to be fluent in English, which can be demonstrated in the interviews and in the English test." ]
+      , isVisible = False
+      }
+    , { question = "Will I need to take an English test?"
+      , answer = [ Paragraph "Yes, for immigration and licensing you will need to take one of either the IELTS or TOEFL english tests unless your nursing education was in the following countries and your textbooks were in English (United Kingdom, Australia, Canada (except Quebec), New Zealand, Ireland or the United States). If you have already taken an English test recently, we can use those scores." ]
+      , isVisible = False
+      }
+    , { question = "What is a passing score for the English test?"
+      , answer = [ Paragraph "For the IELTS (Academic Format) the overall score must be at least 6.5 and the spoken band 7.0. For the TOEFL iBT you must have an overall score greater than 83 and 26 in the speaking section." ]
+      , isVisible = False
+      }
+    , { question = "Will I need a passport?"
+      , answer = [ Paragraph "Yes you and every family member coming with you will need a passport." ]
+      , isVisible = False
+      }
+    , { question = "Will I be able to study and continue to specialize in certain nursing areas after I come?"
+      , answer = [ Paragraph "The US has lots of programs to help you achieve your career goals. Be sure to let your prospective employer know what your goals are." ]
+      , isVisible = False
+      }
+    ]
+        -- Generating id for faq.id
+        |> (\faqs_ -> List.map2 (\id faq -> Faq id faq.question faq.answer faq.isVisible) (List.range 0 (List.length faqs_)) faqs_)
```

Comment:
Nitpicking: you could use [indexedMap](https://package.elm-lang.org/packages/elm/core/latest/List#indexedMap) instead of generating a range.

Reply:
Yes I should have used that :) Thanks!